tests/plugins/test_koji_promote.py

```diff
@@ -731,6 +735,7 @@ def validate_output(self, output, metadata_only):
                 'id',
                 'repositories',
                 'tags',
+                'config'
```

Comment:
Yeah, that's a good idea. I'm gonna add one after some testing on actual docker 1.9 / 1.10 environments